src/components/control-bar/SeekBar.js

```diff
@@ -83,13 +85,36 @@ export default class SeekBar extends Component {
     actions.replay(5);
   }
 
+  getDefaultChildren() {
+    const {
+      player: { currentTime, seekingTime, duration, buffered },
+      mouseTime
+    } = this.props;
+
+    const time = seekingTime || currentTime;
+
+    return [
+      <LoadProgressBar
+        buffered={buffered}
+        currentTime={time}
+        duration={duration}
+      />,
+      <MouseTimeDisplay duration={duration} mouseTime={mouseTime} />,
+      <PlayProgressBar currentTime={time} duration={duration} />
```

Comment:
Non-blocking feedback. Perhaps not strictly necessary at the moment, but I'd recommend passing `key` props to each of these. If this returned array is ever used directly to provide a node for react to render, that'll be useful / important. And I see that we're doing that elsewhere in similar code: https://github.com/newsela/video-react/blob/094d3e18d7f8c477a2704f5fdacdbb6d9f6aa5ec/src/components/Player.js#L117-L136